doc/tags.yml

```diff
@@ -325,7 +294,6 @@ mapping_topics:
                                                      "nrf7000"]
```

Comment:
We can also add tags for https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/networking/softap_wifi_provision.html and https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/networking/wifi_ready.html